math/fastor/include/algebra/math/impl/fastor_transform3.hpp

```diff
@@ -0,0 +1,234 @@
+/** Algebra plugins, part of the ACTS project
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/math/common.hpp"
```

Comment:
Could you please check this include, too?

Reply:
I removed it.